pkg/interceptors/annotater/test-fixtures/deployment-golden.json

```diff
@@ -1,6 +1,10 @@
 {
     "metadata": {
+        "name": "deployment",
```

Comment:
why not call it `type` or `kind`?

Reply:
`deployment` is a bad example here. That is the name of the manifest, which would be something like `customer-silo`.